monix-catnap/shared/src/main/scala/monix/catnap/CircuitBreaker.scala

```diff
@@ -622,7 +632,7 @@ object CircuitBreaker extends CircuitBreakerDocs {
     exponentialBackoffFactor: Double = 1.0,
     maxResetTimeout: Duration = Duration.Inf,
     padding: PaddingStrategy = NoPadding
-  )(implicit F: Sync[F], clock: Clock[F]): F[CircuitBreaker[F]] = {
+  )(implicit F: Sync[F], F2: MonadCancelThrow[F], clock: Clock[F]): F[CircuitBreaker[F]] = {
```

Comment:
I need to revisit this - since I started porting on RC1, Sync now extends MonadCancelThrow.